syntax = "proto3";
package jio.brain.proto.knowledge.healthcare;

import "jio/brain/proto/knowledge/healthcare/base/status.proto";

message GetContextValueNameRequest {
    uint32 context_id = 1;
    uint32 value_id = 2;
}

message GetContextValueNameResponse {
    string context_value_name = 1;
    jio.brain.proto.knowledge.healthcare.base.Status status = 2;
}